src/cap.rs

```diff
@@ -167,4 +167,5 @@ pub enum Cap {
     ArmPtrAuthGeneric = KVM_CAP_ARM_PTRAUTH_GENERIC,
     #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
     X86UserSpaceMsr = KVM_CAP_X86_USER_SPACE_MSR,
+    ExitHypercall = KVM_CAP_EXIT_HYPERCALL,
```

Comment:
I think this is only defined for x86_64 and arm64. From [kvm-bindings](https://github.com/rust-vmm/kvm-bindings):

```
$ git grep KVM_CAP_EXIT_HYPERCALL
src/arm64/bindings.rs:pub const KVM_CAP_EXIT_HYPERCALL: u32 = 201;
src/x86_64/bindings.rs:pub const KVM_CAP_EXIT_HYPERCALL: u32 = 201;
```

Reply:
Interestingly, the docs say this is only available for x86(-64):
https://docs.kernel.org/virt/kvm/api.html#kvm-cap-exit-hypercall

Reply:
Isn't that because `kvm-bindings` only supports x86_64 and arm64? I can't find definitions for any `KVM_CAP_*` outside of those two files.

AFAICT [`KVM_CAP_EXIT_HYPERCALL`](https://github.com/torvalds/linux/blob/55027e689933ba2e64f3d245fb1ff185b3e7fc81/include/uapi/linux/kvm.h#L885) is defined in a non-architecture-specific header.

That said KVM currently only implements `KVM_CAP_EXIT_HYPERCALL` on x86_64, so I'd also be fine with restricting it to that platform.

Reply:
> Interestingly, the docs say this is only available for x86(-64): https://docs.kernel.org/virt/kvm/api.html#kvm-cap-exit-hypercall

If that's what to official docs say, that seems like a good reason to restrict it to x86_64 :)

Reply:
> Isn't that because `kvm-bindings` only supports x86_64 and arm64? I can't find definitions for any `KVM_CAP_*` outside of those two files.

Ah yes, that could be it.